src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -67,7 +67,7 @@ class ValidationStep extends React.Component {
         const errors = {};
 
         _.each(values, (value, key) => {
-            const filteredValue = this.filterInput(value);
+            const filteredValue = typeof value === 'string' ? this.filterInput(value) : value;
```

Comment:
`filterInput` is supposed to work only with string parameters, this avoid passing unexpected values.